org.eclipse.scout.rt.shared/src/main/java/org/eclipse/scout/rt/shared/services/common/context/SharedVariableMap.java

```diff
@@ -73,17 +73,18 @@ public synchronized void updateInternal(Map<String, Object> newMap) {
     putAll(newMap); // fires a value changed event
   }
 
-  private void fireValuesChanged() {
-    m_propertySupport.firePropertyChange(PROP_VALUES, null, CollectionUtility.copyMap(m_variables));
+  private void fireValuesChanged(Map<String, Object> oldVariables) {
+    m_propertySupport.firePropertyChange(PROP_VALUES, oldVariables, CollectionUtility.copyMap(m_variables));
```

Comment:
Add unit test for this new behavior of including oldValues in property change events.